DESCRIPTION

```diff
@@ -91,7 +91,7 @@ Imports:
     withr,
     commonmark (>= 1.7),
     glue (>= 1.3.2),
-    bslib (>= 0.3.0),
+    bslib (>= 0.6.1.9001),
```

Comment:
Will need to add a `Remotes:` field

Reply:
After moving `bind_task_to_button` to `bslib::bind_task_button`, the version requirement goes away, so no need for a Remotes.